CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,3 @@
-jQuery is a [JS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
+jQuery is an [OpenJS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
```

Comment:
The URL should also change to openjsf.org. It redirects there anyway.

Reply:
Fixed. I also updated the Code of Conduct link.

Now, there's one remaining mention of js.foundation left, it's the CLA link: https://cla.js.foundation/jquery/jquery. I think that one needs to stay until we get switched to a different CLA checker.

Reply:
agreed